src/App/Command/GithubCheckIssueCommand.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+
+namespace Console\App\Command;
+
+use DateInterval;
+use DateTime;
+use Console\App\Service\Github;
+use Github\ResultPager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GithubCheckIssueCommand extends Command
+{
+    const LABEL_DUPLICATE = 'label:Duplicate';
+
+    /**
+     * @var Github;
+     */
+    protected $github;
+
+    protected function configure()
+    {
+        $this->setName('github:check:issue')
+            ->setDescription('Check Github Issue')
+            ->addOption(
+                'ghtoken',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                '',
+                $_ENV['GH_TOKEN']
+            )
+            ->addOption(
+                'request',
+                null,
+                InputOption::VALUE_OPTIONAL
+            )
+            ->addOption(
+                'filter:file',
+                null,
+                InputOption::VALUE_OPTIONAL
+            );
+
+    }
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->github = new Github($input->getOption('ghtoken'));
+        $time = time();
+
+        $date = new DateTime();
+        $date->sub(new DateInterval('P1D'));
+        $requests = [
+            // Check Issue created and closed
+            //'Created Issue' => 'is:closed created:>='.$date->format('Y-m-d'),
+            'Created Issue' => 'is:issue created:>=2020-03-11',
```

Comment:
You should calculate this date automatically. If it's 4 weeks in the past, it's easy to do. How do you choose it ?